src/doc-printer.js

```diff
@@ -36,6 +36,17 @@ function makeAlign(ind, n) {
     };
   }
 
+  if (isNaN(n) && n.forceSpace) {
```

Comment:
Please, I would really prefer that this was a Number: `&& !isNaN(n.forceSpaces)`